js/ui/grid_core/ui.grid_core.editing_form_based.js

```diff
@@ -289,6 +289,7 @@ export const editingFormBasedModule = {
                     this._rowsView.renderTemplate($container, template, cellOptions, !!$container.closest(getWindow().document).length).done(() => {
                         this._rowsView._updateCell($container, cellOptions);
                     });
+                    this._rowsView.renderDelayedTemplates();
```

Comment:
In this place renderDelayedTemplates is called in form editing mode. But the problem is actual for popup editing mode only. Also, renderDelayedTemplates should be called once after render, but now it is called after rendering each column with renderAsync.

The better place to call renderDelayedTemplates is here: https://github.com/DevExpress/DevExtreme/blob/22_1/js/ui/grid_core/ui.grid_core.editing_form_based.js#L203